prog/dftb+/lib_dftbplus/initprogram.F90

```diff
@@ -2102,7 +2150,7 @@ subroutine initProgramVariables(input, env)
         ! triplet or spin-polarised
         do iSp = 1, nType
           homoLoc = maxloc(atomEigVal(:orb%nShell(iSp), iSp),&
-              & mask=input%slako%skOcc(:orb%nShell(iSp), iSp) > 0.0_dp)
+              & mask=referenceN0(:orb%nShell(iSp), iSp) > 0.0_dp)
```

Comment:
Same here, can `referenceN0` be used as substitute for the Slater–Koster specific field?